tests/test_cmd_resetdep.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from six import StringIO
+
+from doit.cmd_resetdep import ResetDep
+from doit.dependency import TimestampChecker
+from doit.exceptions import InvalidCommand
+from doit.task import Task
+from tests.conftest import tasks_sample, CmdFactory
+
+
+class TestCmdResetDep(object):
+
+    def test_execute(self, pdepfile, dependency1):
```

Comment:
Why do you think it is required to execute these tests with all backends?
I guess just one should be enough...